.infra/crons.ts

```diff
@@ -42,6 +42,10 @@ export const crons: Cron[] = [
   },
   {
     name: 'personalized-digest',
-    schedule: '15 0 * * *'
-  }
+    schedule: '15 0 * * *',
+  },
+  {
+    name: 'generate-invites',
+    schedule: '15 0 * * *',
```

Comment:
We might want to run it on a different schedule to not block to many things at once.

Reply:
Ideally, at least one hour after the digest